setup.py

```diff
@@ -8,7 +8,7 @@
 # Environment-specific dependencies.
 extras = {
     'surfacecode': ['tensorflow==1.14', 'keras==2.2.4'],
-    'teleportation': []
+    'teleportation' : []
```

Comment:
What's this for?

Reply:
In essence this doesn't have to be there, as the teleportation environment has no additional dependencies. I left it in there for now though to make it clear that this is where the environment specific dependencies go. I suggest deleting the empty teleportation entry once we have another environment with additional dependencies.